### ISSUE TITLE
setup has both android and ios as required, it should be optional

### ISSUE THREAD

**Issue by pankajvaghelav2** [react-native-webrtc/react-native-callkeep#498]
# Bug report

- [x] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

- Reproduced on:
 - [ ] Android
 - [x] iOS

## Description

setup has both android and ios configs as required, it should be optional but if i am only building for one platform,  i shouldnt have to pass settings for other platforms.

it would be helpful if i can only 

## Steps to Reproduce

in RNCallKeep.setup method pass only ios configuration

## Versions
    - Callkeep: 4.2.0
    - React Native: 0.64
    - iOS: 12.5
    - Android:
    - Phone model: 

## Logs

```
Paste here
```

**Comment by m4r00p** [comment]
Nope, actually i will check API level it seems that getDevices appeared on 23 where react native is supporting 21 so for those below 23. We decided to just switch between phone / speaker.

So react-native-callkeep will crash for versions below 23... (You can fix it or just skip it)
